package.json

```diff
@@ -109,5 +115,8 @@
         "react": "^18.2.0",
         "react-dom": "^18.2.0"
     },
-    "dependencies": {}
+    "dependencies": {
```

Comment:
Let's move this to `devDependencies`.
The bundle of `document-model-libs` has to contain all the dependencies codes. It will eventually be imported dynamically on the runtime in Connect, so we can't be sure the necessary dependencies will be included.
An alternative in the future will be for each editor to be a microfontend: https://github.com/originjs/vite-plugin-federation

Reply:
added fix here: [598b3fc](https://github.com/powerhouse-inc/document-model-libs/pull/67/commits/598b3fcb30f47bd532215dd48f6e126b16abe1c7)